mne/annotations.py

```diff
@@ -822,9 +971,17 @@ def set_annotations(self, annotations, on_missing="raise", *, verbose=None):
             self._annotations = new_annotations
         return self
 
-    def get_annotations_per_epoch(self):
+    def get_annotations_per_epoch(self, with_extras=False):
```

Comment:
I think `get_annotations_per_epch(True)` isn't verbose enough, so let's do
```suggestion
    def get_annotations_per_epoch(self, *, with_extras=False):
```